javascript/javascript1/week1/Houseypricey.js

```diff
@@ -0,0 +1,28 @@
+//peter house calculation
+var width = 8;
+var height = 10;
+var depth = 10;
+var gardenSizeInM2 = 100;
+var actualhousePrise = 2500000;
+var volumeInMeters;
+var housePrice;
+volumeInMeters = width * height * depth;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+if (housePrice > actualhousePrise) {
+console.log ("peter is paying exess price");}
```

Comment:
here you can use next line after semi-colon (;) to close the brace ( } ) for neat code.